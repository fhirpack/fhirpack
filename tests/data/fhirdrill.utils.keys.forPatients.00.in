{
  "resourceType": "Patient",
  "id": "525",
  "meta": {
    "versionId": "1",
    "lastUpdated": "2020-06-11T15:27:17.617+00:00",
    "source": "#7IoB9QGABhdzaNv2",
    "tag": [ {
      "system": "https://smarthealthit.org/tags",
      "code": "synthea-5-2019"
    } ]
  },
  "text": {
    "status": "generated",
    "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Generated by <a href=\"https://github.com/synthetichealth/synthea\">Synthea</a>.Version identifier: v2.4.0-100-g26a4b936\n .   Person seed: -4375782139830052088  Population seed: 1559319163074</div>"
  },
  "extension": [ {
    "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-race",
    "extension": [ {
      "url": "ombCategory",
      "valueCoding": {
        "system": "urn:oid:2.16.840.1.113883.6.238",
        "code": "2106-3",
        "display": "White"
      }
    }, {
      "url": "text",
      "valueString": "White"
    } ]
  }, {
    "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity",
    "extension": [ {
      "url": "ombCategory",
      "valueCoding": {
        "system": "urn:oid:2.16.840.1.113883.6.238",
        "code": "2186-5",
        "display": "Not Hispanic or Latino"
      }
    }, {
      "url": "text",
      "valueString": "Not Hispanic or Latino"
    } ]
  }, {
    "url": "http://hl7.org/fhir/StructureDefinition/patient-mothersMaidenName",
    "valueString": "Fleta Bergstrom"
  }, {
    "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-birthsex",
    "valueCode": "F"
  }, {
    "url": "http://hl7.org/fhir/StructureDefinition/patient-birthPlace",
    "valueAddress": {
      "city": "Boston",
      "state": "Massachusetts",
      "country": "US"
    }
  }, {
    "url": "http://synthetichealth.github.io/synthea/disability-adjusted-life-years",
    "valueDecimal": 9.195822017359179
  }, {
    "url": "http://synthetichealth.github.io/synthea/quality-adjusted-life-years",
    "valueDecimal": 18.804177982640823
  } ],
  "identifier": [ {
    "system": "https://github.com/synthetichealth/synthea",
    "value": "4c875c3c-b4d6-4f6d-aabe-5ddc24892adc"
  }, {
    "type": {
      "coding": [ {
        "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
        "code": "MR",
        "display": "Medical Record Number"
      } ],
      "text": "Medical Record Number"
    },
    "system": "http://hospital.smarthealthit.org",
    "value": "4c875c3c-b4d6-4f6d-aabe-5ddc24892adc"
  }, {
    "type": {
      "coding": [ {
        "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
        "code": "SS",
        "display": "Social Security Number"
      } ],
      "text": "Social Security Number"
    },
    "system": "http://hl7.org/fhir/sid/us-ssn",
    "value": "999-99-5015"
  }, {
    "type": {
      "coding": [ {
        "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
        "code": "DL",
        "display": "Driver's License"
      } ],
      "text": "Driver's License"
    },
    "system": "urn:oid:2.16.840.1.113883.4.3.25",
    "value": "S99970944"
  }, {
    "type": {
      "coding": [ {
        "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
        "code": "PPN",
        "display": "Passport Number"
      } ],
      "text": "Passport Number"
    },
    "system": "http://standardhealthrecord.org/fhir/StructureDefinition/passportNumber",
    "value": "X4940908X"
  } ],
  "name": [ {
    "use": "official",
    "family": "Tremblay",
    "given": [ "Addie" ],
    "prefix": [ "Mrs." ]
  }, {
    "use": "maiden",
    "family": "Wolff",
    "given": [ "Addie" ],
    "prefix": [ "Mrs." ]
  } ],
  "telecom": [ {
    "system": "phone",
    "value": "555-856-9091",
    "use": "home"
  } ],
  "gender": "female",
  "birthDate": "1990-05-29",
  "address": [ {
    "extension": [ {
      "url": "http://hl7.org/fhir/StructureDefinition/geolocation",
      "extension": [ {
        "url": "latitude",
        "valueDecimal": 42.115454
      }, {
        "url": "longitude",
        "valueDecimal": -72.539978
      } ]
    } ],
    "line": [ "586 Jacobs Trailer" ],
    "city": "Springfield",
    "state": "Massachusetts",
    "postalCode": "01013",
    "country": "US"
  } ],
  "maritalStatus": {
    "coding": [ {
      "system": "http://terminology.hl7.org/CodeSystem/v3-MaritalStatus",
      "code": "M",
      "display": "M"
    } ],
    "text": "M"
  },
  "multipleBirthBoolean": false,
  "communication": [ {
    "language": {
      "coding": [ {
        "system": "urn:ietf:bcp:47",
        "code": "en-US",
        "display": "English"
      } ],
      "text": "English"
    }
  } ]
}
