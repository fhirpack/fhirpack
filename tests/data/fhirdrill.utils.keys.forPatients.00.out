["resourceType", "id", "meta", "meta.versionId", "meta.lastUpdated", "meta.source", "meta.tag", "meta.tag.system", "meta.tag.code", "text", "text.status", "text.div", "extension", "extension.url", "extension.extension", "extension.extension.url", "extension.extension.valueCoding", "extension.extension.valueCoding.system", "extension.extension.valueCoding.code", "extension.extension.valueCoding.display", "extension.extension.url", "extension.extension.valueString", "extension.url", "extension.extension", "extension.extension.url", "extension.extension.valueCoding", "extension.extension.valueCoding.system", "extension.extension.valueCoding.code", "extension.extension.valueCoding.display", "extension.extension.url", "extension.extension.valueString", "extension.url", "extension.valueString", "extension.url", "extension.valueCode", "extension.url", "extension.valueAddress", "extension.valueAddress.city", "extension.valueAddress.state", "extension.valueAddress.country", "extension.url", "extension.valueDecimal", "extension.url", "extension.valueDecimal", "identifier", "identifier.system", "identifier.value", "identifier.type", "identifier.type.coding", "identifier.type.coding.system", "identifier.type.coding.code", "identifier.type.coding.display", "identifier.type.text", "identifier.system", "identifier.value", "identifier.type", "identifier.type.coding", "identifier.type.coding.system", "identifier.type.coding.code", "identifier.type.coding.display", "identifier.type.text", "identifier.system", "identifier.value", "identifier.type", "identifier.type.coding", "identifier.type.coding.system", "identifier.type.coding.code", "identifier.type.coding.display", "identifier.type.text", "identifier.system", "identifier.value", "identifier.type", "identifier.type.coding", "identifier.type.coding.system", "identifier.type.coding.code", "identifier.type.coding.display", "identifier.type.text", "identifier.system", "identifier.value", "name", "name.use", "name.family", "name.given", "name.prefix", "name.use", "name.family", "name.given", "name.prefix", "telecom", "telecom.system", "telecom.value", "telecom.use", "gender", "birthDate", "address", "address.extension", "address.extension.url", "address.extension.extension", "address.extension.extension.url", "address.extension.extension.valueDecimal", "address.extension.extension.url", "address.extension.extension.valueDecimal", "address.line", "address.city", "address.state", "address.postalCode", "address.country", "maritalStatus", "maritalStatus.coding", "maritalStatus.coding.system", "maritalStatus.coding.code", "maritalStatus.coding.display", "maritalStatus.text", "multipleBirthBoolean", "communication", "communication.language", "communication.language.coding", "communication.language.coding.system", "communication.language.coding.code", "communication.language.coding.display", "communication.language.text"]

